doc/api/util.md

```diff
@@ -126,6 +126,10 @@ util.format(1, 2, 3); // '1 2 3'
 
 ## util.inherits(constructor, superConstructor)
 
+_Note: usage of `util.inherits()` is discouraged. Please use ES6 `class` and
+`extends` to get language level inheritance support, including an unbroken
+inheritance chain._
```

Comment:
Yes, that's right.  `util.inherits()` currently does this:
```js
Object.setPrototypeOf(Desc.prototype, Base.prototype);
```
But to implement inheritance properly, it would have to work like this:
```js
function Base() {
  // ...
}

function Desc() {
  const that = new Base();
  Object.setPrototypeOf(that, Desc.prototype);
  // ...
  return that;
}

Object.setPrototypeOf(Desc, Base);
Desc.prototype = Object.create(Base.prototype, {constructor: {value: Desc}});
```
The boilerplate in `Desc` is approximately equivalent to:
```js
if (!(this instanceof Desc)) return new Desc();
```